src/__tests__/ansiCommandStream.test.js

```diff
@@ -0,0 +1,45 @@
+import hasYarn from 'has-yarn';
```

Comment:
the package 'has-yarn' doesn't export any method called ```__setHasYarn```, i think you should import from ```./has-yarn'```

Reply:
This is because of the mocking in Jest

Reply:
I was wrong, i have just read the documentation of how mocks work...